frontend/tests/ControllerTestCase.php

```diff
@@ -608,7 +608,15 @@ public function getGraderResourcePassthru(
         string $filename,
         bool $missingOk = false
     ): ?bool {
-        throw new \OmegaUp\Exceptions\UnimplementedException();
+        $path = "{$run->run_id}/{$filename}";
+        if (!array_key_exists($path, $this->_resources)) {
+            if (!$missingOk) {
+                throw new \Exception("Resource {$path} not found");
+            }
+            return null;
+        }
+
+        return $this->_resources[$path];
```

Comment:
la idea del passthru es que debería mandar los contenidos del archivo a stdout. tons esto debería ser

```suggestion
        $out = fopen('php://output', 'w');
        fputs($out, $this->_resources[$path]);
        fclose($out);
        return true;
```

en una prueba hay manera de obtener esto sin necesidad de cirugía mayor:

```php
ob_start();
// Manda llamar al API que eventualmente hace passthru
$zipContents = ob_get_contents();
ob_end_clean();
```

Reply:
Listo